src/status_im/ui/screens/contacts/events.cljs

```diff
@@ -4,15 +4,14 @@
             [status-im.data-store.contacts :as contacts]
             [clojure.string :as s]
             [status-im.protocol.core :as protocol]
+            [status-im.utils.contacts :as contact-utils]
```

Comment:
sorry for verbalism, the convention is `:as utils.contacts`

Reply:
cool, will push that and rebase